zerver/webhooks/facebook/doc.md

```diff
@@ -0,0 +1,82 @@
+{!create-stream.md!}
+
+<!--
+{!create-bot-construct-url.md!}
+-->
+
+Next, on your {{ settings_html|safe }},
+[create a bot](/help/add-a-bot-or-integration) for
+{{ integration_display_name }}. Make sure that you select
+**Incoming webhook** as the **Bot type**:
+
+![](/static/images/help/bot_types.png)
+
+The API key for an incoming webhook bot cannot be used to read messages out
+of Zulip. Thus, using an incoming webhook bot lowers the security risk of
+exposing the bot's API key to a third-party service.
+
+Construct the URL for the {{ integration_display_name }}
+bot using the bot's API key and the desired stream name:
+
+`{{ api_url }}{{ integration_url }}?api_key=abcdefgh&stream={{ recommended_stream_name }}&token=sampletoken`
+
+Modify the parameters of the URL above, where `api_key` is the API key
+of your Zulip bot, and `stream` is the stream name you want the
+notifications sent to, and `token` will be used for server verification later,
+please decide your favorite character string.
+
```

Comment:
Make `token will be used for server....` a separate sentence. :)

Reply:
Also, make it `please choose your favorite character string.` And could you please explain as to what you mean by this? Maybe elaborate a little more?

Reply:
@eeshangarg 
I tried to think of it as a friend at school, but I did not know how to fix it.

Would you please tell me how to fix it?